topics/modules-api-ref.md

```diff
@@ -7248,6 +7289,59 @@ Here is a list of events you can use as 'eid' and related sub events:
 
         ValkeyModuleKey *key;    // Key name
 
+* `ValkeyModuleEvent_AuthenticationAttempt`
+
+    Called when an authentication attempt is made, either successful or not.
+
+    The data pointer can be casted to a ValkeyModuleAuthenticationInfo
+    structure with the following fields:
+
+        uint64_t client_id;      // Client ID.
+        const char *username;    // Username used for authentication.
+        const char *module_name; // Name of the module that is handling the
+                                 // authentication. It is NULL if the
+                                 // authentication is handled by the core.
+        ValkeyModuleAuthenticationResult result;   // Result of the authentication:
+                                                   // VALKEYMODULE_AUTH_RESULT_GRANTED or
+                                                   // VALKEYMODULE_AUTH_RESULT_DENIED
+
+* `ValkeyModuleEvent_AtomicSlotMigration`
+
+   Called when an atomic slot migration (CLUSTER MIGRATESLOTS) is started or
+   ended in this node. This node may be a target or a source node, or the
+   target or source might be this node's primary. The following sub events
+   are available:
+
+    * `VALKEYMODULE_SUBEVENT_ATOMIC_SLOT_MIGRATION_IMPORT_STARTED`
+    * `VALKEYMODULE_SUBEVENT_ATOMIC_SLOT_MIGRATION_EXPORT_STARTED`
+    * `VALKEYMODULE_SUBEVENT_ATOMIC_SLOT_MIGRATION_IMPORT_ABORTED`
+    * `VALKEYMODULE_SUBEVENT_ATOMIC_SLOT_MIGRATION_EXPORT_ABORTED`
+    * `VALKEYMODULE_SUBEVENT_ATOMIC_SLOT_MIGRATION_IMPORT_COMPLETED`
+    * `VALKEYMODULE_SUBEVENT_ATOMIC_SLOT_MIGRATION_EXPORT_COMPLETED`
+
+   The data pointer can be casted to `ValkeyModuleAtomicSlotMigrationInfo`
+   structure with the following fields:
+
+        char *job_name;                     // Unique ID for the operation (40 chars)
+        ValkeyModuleSlotRange *slot_ranges; // Array of slot ranges involved in the operation
+        uint32_t num_slot_ranges;           // Number of slot ranges in slot_ranges array
+        char *source_node_id;               // Source node ID (40 chars)
+        char *target_node_id;               // Target node ID (40 chars)
```

Comment:
Whoops, forgot to update these, we no longer include source/target node: https://github.com/valkey-io/valkey/blob/unstable/src/valkeymodule.h#L831-L836